### PR TITLE
Solar Panels

### PR DESCRIPTION
**Pull request by poggers16**
#551 restored :3
[REDACTED]

### CODE REVIEW

File: common/src/main/java/com/gregtechceu/gtceu/api/cover/CoverBehavior.java [hunk 1]
```diff
@@ -27,8 +27,10 @@
 import net.minecraft.world.level.block.state.BlockState;
 import net.minecraft.world.phys.BlockHitResult;
 
+import javax.annotation.Nonnull;
 import javax.annotation.Nullable;
 import javax.annotation.ParametersAreNonnullByDefault;
+import java.security.DrbgParameters;
```
Comment (screret) [inline common/src/main/java/com/gregtechceu/gtceu/api/cover/CoverBehavior.java:33]: tf?

File: common/src/main/java/com/gregtechceu/gtceu/utils/GTUtil.java [hunk 2]
```diff
@@ -9,16 +10,21 @@
...
 import com.lowdragmc.lowdraglib.side.fluid.FluidTransferHelper;
 import com.lowdragmc.lowdraglib.side.fluid.IFluidTransfer;
+import com.mojang.authlib.minecraft.client.MinecraftClient;
+import net.minecraft.client.Minecraft;
 import com.mojang.blaze3d.platform.InputConstants;
 import dev.architectury.injectables.annotations.ExpectPlatform;
-import net.minecraft.client.Minecraft;
+import io.github.fabricators_of_create.porting_lib.util.MinecraftClientUtil;
+import net.minecraft.world.level.Level;
+import net.minecraft.core.BlockPos;
```
Comment (screret) [inline common/src/main/java/com/gregtechceu/gtceu/utils/GTUtil.java]: tf are these imports?

File: common/src/main/java/com/gregtechceu/gtceu/api/cover/CoverBehavior.java (hunk 1 above)
Comment (screret) [inline common/src/main/java/com/gregtechceu/gtceu/api/cover/CoverBehavior.java]: same here

**Review by screret — CHANGES_REQUESTED** [review]

File: common/src/main/java/com/gregtechceu/gtceu/api/cover/CoverBehavior.java (hunk 1 above)
Comment (poggers16) [reply common/src/main/java/com/gregtechceu/gtceu/api/cover/CoverBehavior.java]: removed all of those

File: common/src/main/java/com/gregtechceu/gtceu/utils/GTUtil.java (hunk 2 above)
Comment (poggers16) [reply common/src/main/java/com/gregtechceu/gtceu/utils/GTUtil.java]: removed those imports 🥺

**Review by screret — APPROVED** [review]
don't merge for this update, but next one.

**Pull request merged**